scripts/generate_babs_dic.py

```diff
@@ -0,0 +1,23 @@
+import json
+
+import pandas as pd
```

Comment:
I have now used pandas to read the excel file but that package doesn't seem to be "official" even though I could only find pandas used as a tool to read excel files on github. Should the linter ignore this file?